source/config/Config.h

```diff
@@ -102,9 +104,20 @@ namespace Aws
                     static constexpr char JSON_KEY_LOG_TYPE[] = "type";
                     static constexpr char JSON_KEY_LOG_FILE[] = "file";
 
-                    int logLevel{3};
-                    std::string type;
-                    std::string file;
+                    static constexpr char CLI_ENABLE_SDK_LOGGING[] = "--enable-sdk-logging";
+                    static constexpr char CLI_SDK_LOG_LEVEL[] = "--sdk-log-level";
+                    static constexpr char CLI_SDK_LOG_FILE[] = "--sdk-log-file";
+
+                    static constexpr char JSON_KEY_ENABLE_SDK_LOGGING[] = "enable-sdk-logging";
+                    static constexpr char JSON_KEY_SDK_LOG_LEVEL[] = "sdk-log-level";
+                    static constexpr char JSON_KEY_SDK_LOG_FILE[] = "sdk-log-file";
+
+                    int deviceClientlogLevel{3};
+                    std::string deviceClientLogtype;
+                    std::string deviceClientLogFile;
+                    bool sdkLoggingEnabled = false;
+                    Aws::Crt::LogLevel sdkLogLevel = Aws::Crt::LogLevel::Trace;
+                    std::string sdkLogFile = "/var/log/aws-iot-device-client/sdk.log";
```

Comment:
Create a Default_Path constant over [here](https://github.com/awslabs/aws-iot-device-client/blob/main/source/config/Config.h#L220-L225) to save path and use it in DC application code and in unit tests.

Reply:
That's a good idea! I have created the constant, but rather than placing it at the region of Config.h that you specified, I placed it in the PlainConfig::LogConfig namespace

Reply:
No problem. We should consider moving other defaults in respective "PlainConfig::.." namespaces. But that change is for a different PR.